marklogic-data-hub-central/ui/src/components/detail-page-non-entity/detail-page-non-entity.tsx

```diff
@@ -259,8 +294,39 @@ const DetailPageNonEntity = (props) => {
                 data-testid="history-table"
               />
             </div>
+            {
+              (props.collections) ?
```

Comment:
Suggestion: 
We can also use (props.collections) && Condition 2 in order to avoid ( **: null** )